examples/middleware/multiple_pipelines/src/main.rs

```diff
@@ -73,7 +73,7 @@ impl Middleware for ApiMiddleware {
                     mime::APPLICATION_JSON,
                     body,
                 );
-                Either::B(future::ok((state, response)))
+                Box::new(future::ok((state, response)))
```

Comment:
Is it possible to pass `future::ok((state, response))` here instead of `Box`? I believe `f` is double-boxed on line 79.

Reply:
Thanks @secretfader - I missed that.
The `match` has to return a box since `chain` returns a box in the other arms, but you're right, I had missed the extra box at the end. Opened #306 to tidy that up, since this is merged :)